packages/web/src/components/ui/PackageVersion/PackageVersion.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import styles from './PackageVersion.module.scss';
+import clsx from 'clsx';
+import { Icon } from '../Icon/Icon';
+import CircularProgress from '../CircularProgress/CircularProgress';
+import ProgressBar from '../ProgressBar/ProgressBar';
+import { formatNumber } from 'utils/helpers';
+
+type Module = {
+  path: string;
+  moduleSize: number;
+  moduleSizeUnit: string;
+  percentage: number;
+};
+
+type EntryPoint = {
+  value: number;
+  code: string;
+  entry: string;
+};
+
+type Props = {
+  version: string;
+  updateDate: string;
+  uses: number;
+  size: number;
+  sizeUnitShorthand: string;
+  sizeUnit: string;
+  modulesCount: number;
+  modules: Module[];
+  entries: EntryPoint[];
+  opened?: boolean;
+};
+
+const PackageVersion = ({
```

Comment:
nit: `PackageVersionDetailsBlock` or something along the lines of. Current same suggests more about displaying a package version string like `1.0.0`